ui/components/ui/error-message/error-message.component.js

```diff
@@ -12,14 +12,25 @@ const ErrorMessage = (props, context) => {
         alt=""
         className="error-message__icon"
       />
-      <div className="error-message__text">{error}</div>
+      <div className="error-message__text">
+        {error}
+        {linkText ? (
+          <a className="error-message__link" onClick={onErrorMessageClick}>
```

Comment:
IIRC this needs an `href` to be keyboard focusable. Elsehwere we've been settting `href` to `"#"` if we don't want to navigate anywhere, just to ensure it's focusable.

```suggestion
          <a className="error-message__link" href="#" onClick={onErrorMessageClick}>
```